### PR TITLE
[DO NOT MERGE] Cron initial

### PR DESCRIPTION
**Pull request by SAGV**

### CODE REVIEW

**Comment by SAGV** [comment]
That functionality doesn't work as expected. The issue is that cron doesn't start phantomjs (or kills it right away). Running same "start" command via bash works just fine

File: docker/crons.conf [hunk 1]
```diff
@@ -0,0 +1,2 @@
+* * * * * /bin/bash /usr/bin/restart.sh
+* * * * * echo "Hello world" >> /usr/src/app/phantomjs.log
```
Comment (SAGV) [inline docker/crons.conf:2]: Remove that when done

File: docker/restart.sh [hunk 2]
```diff
@@ -0,0 +1,8 @@
+#!/bin/sh
+# restart.sh
+
+# Kill old phantomjs
+kill $(ps aux | grep '[.]/docker/phantomjs server.js' | awk '{print $2}'); 
+
+# Start new one
+/usr/src/app/docker/phantomjs /usr/src/app/server.js
```
Comment (SAGV) [inline docker/restart.sh:8]: Starting it again doesn't

File: docker/restart.sh [hunk 3]
```diff
@@ -0,0 +1,8 @@
+#!/bin/sh
+# restart.sh
+
+# Kill old phantomjs
+kill $(ps aux | grep '[.]/docker/phantomjs server.js' | awk '{print $2}');
```
Comment (SAGV) [inline docker/restart.sh:5]: Killing works

**Comment by SAGV** [comment]
@jcassee feel free to shoot if you have a solution in mind. I've exceeded the timebox and I'll try once again after we're done with payment flow

File: docker/crons.conf [hunk 4]
```diff
@@ -0,0 +1,2 @@
+* * * * * /bin/bash /usr/bin/restart.sh
```
Comment (SAGV) [inline docker/crons.conf:1]: When `restart.sh` is fixed, change that one to `once in 15 mins`

File: docker/crons.conf (hunk 1 above)
Comment (jcassee) [inline docker/crons.conf:2]: What schedule is `* * * * *`? That is every second, no? Also, you are copying this to `/etc/crons.conf`. I do not recognize the location. Also², why do you think `cron` will run in this container?

Comment (SAGV) [inline docker/crons.conf:2]: Nope, it is every minute. 

There is a command `* * * * * echo "Hello world" >> /usr/src/app/phantomjs.log`which will put `Hello world` into a file every minute. So with that command you will see that the cron actually works. The log will show each time cron has executed something

Comment (jcassee) [inline docker/crons.conf:2]: Okay. Do you have a link to an explanation of `/etc/crons.conf`?

Comment (SAGV) [inline docker/crons.conf:2]: Sure
https://www.pantz.org/software/cron/croninfo.html

Comment (jcassee) [inline docker/crons.conf:2]: Thanks, but that link says nothing about `/etc/crons.conf`. Cron looks at `/etc/crontab`, which in turn often includes `/etc/cron.d/*`, `/etc/cron.daily/*`, etc.

**Pull request closed without merging**